mbed-events/equeue/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2016 Christopher Haster
```

Comment:
Seems there are multiple licenses in mbed-events: Apache (https://github.com/ARMmbed/mbed-events/blob/master/LICENSE) and MIT (https://github.com/ARMmbed/mbed-events/blob/master/equeue/LICENSE). I don't think either is an issue, although I don't understand which are the Apache parts and which are the MIT parts. About the copyright line, I don't know the full history of the mbed-events library. That's something that needs to be sorted with @geky.